src/shared/services/api/api.test.js

```diff
@@ -165,4 +167,19 @@ describe('api service', () => {
       expect(gen.next(token).value).toEqual(call(request, url, postBody));
     });
   });
+
+  describe('getErrorMessage', () => {
+    it('returns a default error message', () => {
+      expect(getErrorMessage({ status: 415 })).toEqual(defaultErrorMessage);
+      expect(getErrorMessage({ status: 'foo bar' })).toEqual(defaultErrorMessage);
+    });
+
+    it('returns a specific error message', () => {
+      const statuses = [401, 403, 408, 413, 429, 500, 503];
+
+      statuses.forEach(status => {
+        expect(getErrorMessage({ status })).not.toEqual(defaultErrorMessage);
```

Comment:
Could we also check that they **DO** receive the correct error message. Not just **NOT** default

Reply:
Done.